modules/cco-short-term-creds-format-aws.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing_cloud_provider_credentials/cco-short-term-creds.adoc
+
+:_content-type: REFERENCE
+[id="cco-short-term-creds-format-aws_{context}"]
+= AWS component secret formats
+
+Using manual mode with STS changes the content of the AWS credentials that are provided to individual {product-title} components.
+
+.AWS secret format using long-term credentials
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: <target-namespace> <1>
+  name: <target-secret-name> <2>
```

Comment:
Check all the user-replaced values in this module to make sure they use underscores:
```suggestion
  namespace: <target_namespace> <1>
  name: <target_secret_name> <2>
```